lib/index.js

```diff
@@ -437,7 +437,7 @@ module.exports.renderSync = function(opts) {
     return result;
   }
 
-  throw util._extend(new Error(), JSON.parse(result.error));
+  throw assign(new Error(), JSON.parse(result.error));
```

Comment:
`throw new Error(JSON.parse(result.error))` in this case throws `Error: [object Object]` instead of intended `{ Error: Oops I did it again`.